pkg/option/config.go

```diff
@@ -1226,13 +1226,19 @@ type IpvlanConfig struct {
 	OperationMode     string
 }
 
+// IPAddrCIDR is a IP format structure used by NAT46Prefix, LoadBalancerRSSv4 and LoadBalancerRSSv6.
+type IPAddrCIDR struct {
+	IP   net.IP
+	Mask string
+}
+
 // DaemonConfig is the configuration used by Daemon.
 type DaemonConfig struct {
 	CreationTime        time.Time
 	BpfDir              string     // BPF template files directory
 	LibDir              string     // Cilium library files directory
 	RunDir              string     // Cilium runtime directory
-	NAT46Prefix         *net.IPNet // NAT46 IPv6 Prefix
+	NAT46Prefix         IPAddrCIDR // NAT46 IPv6 Prefix
```

Comment:
I'm not sure we need to define a new type and replace them here. 

Instead, can we just use the String method on the original *net.IPNet to extract the mask later in code? 

For example:

``` 
	_, net, _ := net.ParseCIDR("192.168.0.128/32")
	idx := strings.IndexRune(net.String(), '/')
	mask := []rune(net.String())[idx:]
	fmt.Printf(string(mask)) // will print /32
```